grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -394,7 +394,7 @@ class Order implements Serializable {
     }
 
     Boolean getHasItemsOrAdjustments() {
```

Comment:
If you return only active this should be indicated in the naming, because this one suggests, that it will check for any kind of order items or adjustments.